### PR TITLE
HPCC-26025 Roxie will restart during startup if file copies take too long

### PR DESCRIPTION
**Pull request by richardkchapman**
Partial fix allowing Roxie startup probe timeouts to be overridden

Signed-off-by: Richard Chapman <rchapman@hpccsystems.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: helm/hpcc/values.schema.json [hunk 1]
```diff
@@ -1453,6 +1465,14 @@
         }
       }
     },
+    "minStartupTime": {
+      "description": "The time to wait before initiating startup probing (in seconds). Default 0",
+      "type": "string"
```
Comment (ghalliday) [inline helm/hpcc/values.schema.json:1470]: I think you mean integer...

**Review by ghalliday — COMMENTED** [review]
@richardkchapman minor problem.  It might be worth adding examples to the test helms as well.

File: helm/hpcc/values.schema.json (hunk 1 above)
Comment (richardkchapman) [reply helm/hpcc/values.schema.json:1470]: Interesting. It didn't complain! I will fix

**Review by ghalliday — APPROVED** [review]

File: helm/hpcc/docs/changes.md [hunk 2]
```diff
@@ -117,3 +117,9 @@ sasha:
     storage:
 ```
 
+# Startup probe changes
+
+Some components may take a while to start up, but should not be added to the relevant k8s load balancer until they have done so. We us
```
Comment (jakesmith) [inline helm/hpcc/docs/changes.md:122]: trivial/typo: us -> use

**Pull request merged**